linux_sql/scripts/psql_docker.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+# script usage
+#./scripts/psql_docker.sh start|stop|create [db_username][db_password]
+
+enter_cmd=$1
+db_username=$2
+db_password=$3
+
+
+#if user enter create option
+if [ "$1" == "create" ] && [ "$#" = "3" ];
+then
+	#check whether container already created
+	if [ "$(docker ps -f name= "jrvs-psql" | wc -l)" = "2" ];
+	then
+		echo "container is already created"
+       	 	exit 1
+		
+	fi
+
+        docker volume create pgdata 
+
+	#create container
+	if [ "$(docker container ls -a -f name= "jrvs-psql" | wc -l)" != "2" ];
+   	then 
+		docker run --name jrvs-psql -e POSTGRES_USER=${db_username} -e POSTGRES_PASSWORD=${db_password} -d -v pgdata:/var/lib/postgresql/data -p 5432:5432 postgres
+		echo "container created"
+		exit 0
+	fi
+
+
+
+	
+
+fi
+
+
+#if user didn't enter username or password
+if [ "$1" == "create" ] && [ "$#"!=3 ];
+then 
+	echo "Invalid argument, please enter username and password"
+        exit 1
+fi
+
+
+#if user enter start option 
+if [ "$1" == "start" ]; 
+then 
+    #start docker 
+    	systemctl status docker && systemctl start docker
```

Comment:
systemctl status docker || systemctl start docker
you need to know why

Reply:
check the status and start docker not or, or means choose one of them